.github/workflows/main.yml

```diff
@@ -147,13 +147,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        launcher: [Chrome, Firefox, IE]
```

Comment:
we should still test IE, just not IE11. I believe Edge comes installed on these VMs.